maven/bnd-export-maven-plugin/pom.xml

```diff
@@ -26,6 +26,7 @@
 		<dependency>
 			<groupId>org.apache.maven</groupId>
 			<artifactId>maven-plugin-api</artifactId>
+			<scope>compile</scope>
```

Comment:
I would have thought the scope should have been set in the dependenciesManagement section in the parent.  I would also think this should not need to be set here?

Reply:
So, this is needed when you have Mojo inheritance using maven annotations (look it up). But since we don't have that anymore, I can remove it.